src/Featurizers/Components/GrainFeaturizerImpl.h

```diff
@@ -190,6 +190,7 @@ class GrainEstimatorImplBase : public BaseT {
     using InputType                         = typename TheseGrainEstimatorTraits::InputType;
     using TransformedType                   = typename TheseGrainEstimatorTraits::TransformedType;
 
+    using GrainEstimatorAnnotation          = Microsoft::Featurizer::Featurizers::Components::GrainEstimatorAnnotation<GrainType>;

```

Comment:
I don't think this need all prefix namespaces.

Reply:
I remember now - some compilers will issue this error when the name isn't fully qualified:

```
In file included from /home/default/agent/_work/1/s/src/Featurizers/Components/UnitTests/GrainFeaturizerImpl_UnitTest.cpp:8:0:
      /home/default/agent/_work/1/s/src/Featurizers/Components/UnitTests/../GrainFeaturizerImpl.h:193:82: error: declaration of ‘using GrainEstimatorAnnotation = class Microsoft::Featurizer::Featurizers::Components::GrainEstimatorAnnotation<GrainT>’ [-fpermissive]
           using GrainEstimatorAnnotation          = GrainEstimatorAnnotation<GrainType>;
                                                                                        ^
      In file included from /home/default/agent/_work/1/s/src/Featurizers/Components/UnitTests/GrainFeaturizerImpl_UnitTest.cpp:8:0:
      /home/default/agent/_work/1/s/src/Featurizers/Components/UnitTests/../GrainFeaturizerImpl.h:22:7: error: changes meaning of ‘GrainEstimatorAnnotation’ from ‘class Microsoft::Featurizer::Featurizers::Components::GrainEstimatorAnnotation<GrainT>’ [-fpermissive]
       class GrainEstimatorAnnotation : public Annotation {
```